src/libraries/react-calendar/components/drawer/CalendarDrawer.tsx

```diff
@@ -0,0 +1,12 @@
+import CalendarSmall from '../miniature/CalendarSmall'
+import { CalendarSmallWrapper } from './CalendarDrawer.styled'
+
+function CalendarDrawer() {
```

Comment:
I think it would make more sense to remove this component as the only styled component you have should acctually be inside CalendarSmall.